src/Nethermind/Nethermind.Evm/Tracing/GethStyle/Custom/Native/Call/NativeCallTracerCallFrame.cs

```diff
@@ -38,12 +41,12 @@ public class NativeCallTracerCallFrame : IDisposable
 
     public void Dispose()
     {
-        Input?.Dispose();
-        Output?.Dispose();
-        Logs?.Dispose();
-        foreach (NativeCallTracerCallFrame childCallFrame in Calls)
+        if (Interlocked.CompareExchange(ref _disposed, 1, 0) == 0)
```

Comment:
I know it's used once, but still I'd extract `const int Alive = 0` and potentialy dead consts.